### ISSUE TITLE
Quest Borean Tundra : Escape from the winterfin cavern

### ISSUE THREAD

**Issue by FONEY** [wow-mania/Redemption#524]
During the escort quest, the murloc dissapears when he has to cross the water to the questgiver.

**Comment by dimman321** [comment]
Yes he did, but if u wait a little you wouldve finnished the quest, same happened to me aswell :)
